Course_Part_07/src/assignment29/Assignment29.java

```diff
@@ -0,0 +1,20 @@
+package assignment29;
+
+public class Assignment29 {
```

Comment:
Uitwerking van assignment 29 is prima! Samen met de vorigen is duidelijk dat je de materie begrijpt 👍.

Voor de oefening zou je assignment 29 nog eens kunnen doen waarbij je deze punten meeneemt:
- alle fields private
- alle methodes public
- de class Account van assignment 28 gebruiken ipv te kopieren en zijn fields private maken, maar let op dit "sloopt" ook assignment 28.

Zie maar ff of je tijd / zin hebt om ermee bezig te gaan.